src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -976,22 +962,21 @@ static BOOL AttributeHasChangesOmittedQuality(MTRAttributePath * attributePath)
                             if (values) {
                                 // Since the format is the same data-value dictionary, this looks like an
                                 // attribute report
-                                MTR_LOG_INFO("%@ completion values %@", logPrefix, values);
+                                MTR_LOG_INFO("Read attribute result: %@", values);
```

Comment:
A bunch of these logs could use the item id too, right?  Will stop commenting on this.